tests/beaker_tests/cisco_interface/routedintf_provider_nondefaults.rb

```diff
@@ -152,6 +156,10 @@
                                  /speed 100/,
                                  /duplex full/,
                                  /vrf member test1/,
+                                 /ip access-group v4acl1 in/,
```

Comment:
Don't worry about validating show output.. just use the `puppet resource` command to validate changes.